server/data/strapiApiClient.ts

```diff
@@ -136,7 +136,7 @@ export default class StrapiApiClient {
     return this.restClient.get({ path, query })
   }
 
-  async getNamespaces(): Promise<Array<Namespace>> {
+  async getNamespaces(): Promise<Unwrapped<Namespace>[]> {
```

Comment:
This makes sense - we should be exposing the new types from the client but ideally we wouldn't expose this `Unwrapped` type everywhere..